CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 =================================
 
 ## Change Log:
+### Version 4.0.0   *(Date)*
+-------------------------------------------
+* MoEngage-iOS-SDK dependency updated to 9.1.* and MoEngageInApp dependency updated to 4.1.*.
```

Comment:
@RakshithaAcharya Should be 9.2.x I believe

Reply:
@deepa-iOS-SDK  ill do it main branch